client/src/components/About/About.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import CountUp from "react-countup";
+import ScrollTrigger from "react-scroll-trigger";
+
+function About() {
+  //to set state for numbers counting up on scrolling
+  const [counterOn, setCounterOn] = useState(false);
+
+  return (
+    <div>
+      {/* div for the main section */}
+      <div className="relative bg-[#354F52] h-screen">
+        <div className="font-['Raleway'] absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 text-4xl sm:text-6xl text-left text-[#84A98C]">
+          RENAISSANCE <br />
+          <span className=" text-white">7.0</span>
+        </div>
```

Comment:
The following elements are missing as per the design.
![image](https://user-images.githubusercontent.com/77530596/217621326-2c7317c0-4f2d-4513-b138-c145d9be38cb.png)

Reply:
![Screenshot (218)](https://user-images.githubusercontent.com/93890098/217761447-131004d9-2bce-49dc-b122-1bdf5f118a3b.png)
added the missing elements